add-ons/grafana-prometheus/README.md

```diff
@@ -0,0 +1,39 @@
+# Grafana Add-on
+
+This addon installs grafana for metrics visualization from prometheus
+
+## Prerequisites
+
+The prometheus addon is required
+
+```
+minishift addon install prometheus-3.7
+minishift addon apply prometheus-3.7 --addon-env namespace=kube-system
+```
+
+## Deploy grafana
+
+To deploy grafana do:
+
+```
+minishift addon apply grafana --addon-env namespace=grafana
```

Comment:
Looks like not must be 'grafana_namespace' instead of 'namespace' and 'grafana-prometheus' instead of 'grafana'